### PR TITLE
Create deletePlatformSignup route

### PR DESCRIPTION
**Pull request by ya5er**
## Notion ticket link

[Create DELETE route for platformSignups](https://www.notion.so/uwblueprintexecs/Create-DELETE-route-for-platformSignups-85ae09d6fb64475c9f0640b63e52b631?pvs=4)

## Implementation description
* Created delete route for platformSignups
* Deletes a signup given an id

## Steps to test
1. N/A

## What should reviewers focus on?
* Parameter in URL

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: backend/typescript/rest/signupRequestRoutes.ts [hunk 1]
```diff
@@ -17,4 +17,16 @@ platformSignupRouter.get('/getPlatformSignup', isAuthorizedByRole(new Set(["ADMI
         res.status(500).json({ error: getErrorMessage(error) });
     }
 });
-  
\ No newline at end of file
+
+platformSignupRouter.delete('/deletePlatformSignup/:id', isAuthorizedByRole(new Set(["ADMIN"])), async (req, res) => {
```
Comment (sebmendoza) [inline backend/typescript/rest/signupRequestRoutes.ts]: What does Set(["ADMIN"]) do?

**Review by sebmendoza — COMMENTED** [review]
Do you have any screenshots of you testing it in Postman?

**Comment by ya5er** [comment]
Screenshot 2024-02-22 at 9 10 00 PM

**Review by kevinouyang1 — APPROVED** [review]
loooks good!!

**Pull request merged**